### ISSUE TITLE
failing travis functional tests

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#3068]
ref https://travis-ci.org/mozilla/fxa-content-server/jobs/80627103

```
make: Entering directory `/home/travis/build/mozilla/fxa-content-server/node_modules/fxa-auth-server/node_modules/scrypt-hash/build'

  CC(target) Release/obj.target/scrypt/crypto_scrypt-sse.o
  CXX(target) Release/obj.target/scrypt/node_scrypt.o
  SOLINK_MODULE(target) Release/obj.target/scrypt.node
  COPY Release/scrypt.node

make: Leaving directory `/home/travis/build/mozilla/fxa-content-server/node_modules/fxa-auth-server/node_modules/scrypt-hash/build'

fxa-auth-server@1.45.0 node_modules/fxa-auth-server
├── through@2.3.7
├── hkdf@0.0.2
├── uuid@1.4.1
├── openid@0.5.13
├── poolee@1.0.0 (keep-alive-agent@0.0.1)
├── envc@2.2.0 (params@0.1.1)
├── binary-split@0.1.2 (bops@0.0.6)
├── hapi-auth-hawk@2.0.0 (boom@2.8.0, hoek@2.14.0)
├── bluebird@2.9.25
├── fxa-jwtool@0.7.1 (bluebird@2.9.15, fetch@0.3.6)
├── request@2.55.0 (caseless@0.9.0, aws-sign2@0.5.0, forever-agent@0.6.1, stringstream@0.0.4, oauth-sign@0.6.0, tunnel-agent@0.4.1, isstream@0.1.2, json-stringify-safe@5.0.1, node-uuid@1.4.3, qs@2.4.2, combined-stream@0.0.7, form-data@0.2.0, mime-types@2.0.14, http-signature@0.10.1, bl@0.9.4, tough-cookie@2.0.0, har-validator@1.8.0)
├── scrypt-hash@1.1.12 (bindings@1.2.1, nan@1.8.4)
├── aws-sdk@2.1.26 (xmlbuilder@0.4.2, xml2js@0.2.8, sax@0.5.3)
├── joi@6.4.1 (topo@1.0.3, isemail@1.1.1, hoek@2.14.0, moment@2.10.6)
├── hapi@8.4.0 (cryptiles@2.0.4, peekaboo@1.0.0, ammo@1.0.0, accept@1.0.0, topo@1.0.2, items@1.1.0, kilt@1.1.1, catbox-memory@1.1.1, call@2.0.1, qs@2.4.0, shot@1.4.2, boom@2.6.1, iron@2.1.2, wreck@5.2.0, hoek@2.11.1, catbox@4.2.2, mimos@2.0.2, inert@2.1.4, subtext@1.0.2, joi@6.0.8, heavy@3.0.0, statehood@2.0.0, h2o2@4.0.0, vision@2.0.0)
├── convict@1.0.1 (depd@1.0.1, validator@3.43.0, optimist@0.6.1, varify@0.1.1, moment@2.10.6, cjson@0.3.1)
└── fxa-auth-mailer@1.0.8 (jed@0.5.4, rc@0.5.0, bluebird@2.9.34, po2json@0.4.1, bunyan@1.0.0, handlebars@1.3.0, restify@2.8.2, fxa-content-server-l10n@0.0.0, nodemailer@0.7.1)
+cd node_modules/fxa-auth-server
+node ./scripts/gen_keys.js
+LOG_LEVEL=error
+node ./node_modules/fxa-auth-db-mysql/bin/mem.js
module.js:340
    throw err;
          ^
Error: Cannot find module '/home/travis/build/mozilla/fxa-content-server/node_modules/fxa-auth-server/node_modules/fxa-auth-db-mysql/bin/mem.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
Generating keypair
Secret Key saved: /home/travis/build/mozilla/fxa-content-server/node_modules/fxa-auth-server/config/secret-key.json
undefined:0
^
SyntaxError: Unexpected end of input
    at Object.parse (native)
    at /home/travis/build/mozilla/fxa-content-server/node_modules/fxa-auth-server/scripts/gen_keys.js:49:18
    at ChildProcess.exithandler (child_process.js:662:7)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:766:16)
    at Process.ChildProcess._handle.onexit (child_process.js:833:5)
The command "./tests/ci/deps.sh" failed and exited with 8 during .
Your build has been stopped.
```

From @philbooth  in IRC:
```
[13:59:08]  <pb>	so dev deps were removed from shrinkwrap in the auth-server as per this comment:
[13:59:09]  <pb> https://github.com/mozilla/fxa-auth-server/pull/1034#issuecomment-139364145
[14:00:24]  <pb>	which means db-mysql is not being installed when your tests/ci/deps.sh script runs `npm i mozilla/fxa-auth-server`
[14:01:06]  <pb>	i guess for now you can hack the script to force-install db-mysql
[14:01:33]  <pb>	but you should probably speak to danny about whether the dev deps should go back in to shrinkwrap or not
```

cc @dannycoates

**Comment by shane-tomlinson** [comment]
ref #3067

**Comment by jrgm** [comment]
I think npmjs is having a bad day - http://status.npmjs.org/

**Comment by dannycoates** [comment]
fixed #3069
